### PR TITLE
Change visualization in Meetings

### PR DESCRIPTION
**Pull request by LauraJaimeC**
#### :tophat: What? Why?
The visualization meetings has been changed.

For it:
- Add a time filter
- Add a day of the week filter
- The way meetings are seen in the public part has been changed. (now they show in a list)
- A setting has been added in the component to choose how the public part looks. (cards visualization or list)
- The meetings are order by start time 

#### :clipboard: Subtasks
- [ ] Add tests
- [ ] Add documentation regarding the feature 
- [ ] Add/modify seeds
- [ ] Another subtask

### :camera: Screenshots (optional)

- Setting in component

Captura de pantalla de 2021-04-29 13-12-39

- Filters

Captura de pantalla de 2021-04-29 13-13-04

- Meeting list

Captura de pantalla de 2021-04-29 13-13-21

### CODE REVIEW

File: app/decorators/models/decidim/meeting_decorator.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Decidim::Meetings::Meeting.class_eval do
+  scope :thirteen_am, -> { where("start_time::time < ?", "13:00:00") }
```
Comment (tramuntanal) [inline app/decorators/models/decidim/meeting_decorator.rb:4]: We must be aware of the TimeZone and the daylight savings. Better create a Ruby Time object at 13:00 and pass it as parameter.

**Review by tramuntanal — CHANGES_REQUESTED** [review]
@LauraJaimeC great PR!
I've just commented a detail about TimeZones

File: app/decorators/models/decidim/meeting_decorator.rb (hunk 1 above)
Comment (LauraJaimeC) [reply app/decorators/models/decidim/meeting_decorator.rb]: Can you check if it is correct now?

**Review by tramuntanal — DISMISSED** [review]
Good job!

**Review by tramuntanal — APPROVED** [review]
:+1:

**Pull request merged**